plugins/README.md

```diff
@@ -0,0 +1,30 @@
+# LOOP Plugins
+
+:warning: Experimental :warning:
+
+This directory supports Local-Out-Of-Process (LOOP) Plugins, an alternative node runtime where some systems execute in 
+separate processes, plug-in via [github.com/hashicorp/go-plugin](https://github.com/hashicorp/go-plugin), and 
+communicate via [GRPC](https://grpc.io).
+
+There are currently two kinds of plugins, and one implementation of each: a Solana Relayer plugin, and a Median Reporting
+plugin. The [cmd](cmd) directory contains their `package main`s for now. These can be built via `make install-solana` and 
+`make install-median`.
+
+## How to use
+
+[chainlink.Dockerfile](chainlink.Dockerfile) extends the regular [core/chainlink.Dockerfile](../core/chainlink.Dockerfile)
+to include the plugin binaries, and enables support by setting `CL_SOLANA_CMD` and `CL_MEDIAN_CMD`. Either plugin can be
+disabled by un-setting the environment variable. Images built from this file can otherwise be used normally, provided 
```

Comment:
> Either plugin can be disabled by un-setting the environment variable.

Does this read like the system would be disabled completely? Perhaps needs clarity that disabling just means reverting to the original in-process runtime.